sdgym/datasets.py

```diff
@@ -62,22 +63,24 @@ def _get_dataset_path(modality, dataset, datasets_path, bucket=None, aws_key=Non
     return dataset_path
 
 
-def _apply_max_columns_to_metadata(metadata, max_columns):
-    tables = metadata['tables']
-    for table in tables.values():
-        fields = table['fields']
-        if len(fields) > max_columns:
-            fields = dict(itertools.islice(fields.items(), max_columns))
-            table['fields'] = fields
+def _get_dataset_subset(data, metadata_dict):
+    if 'tables' in metadata_dict.keys():
+        raise ValueError('max_columns is not supported for multi-table datasets')
```

Comment:
Should we modify the error message since `max_columns` is no longer a method parameter?